### DISCUSSION TITLE
Welcome to PotatoNV Discussions!

### DISCUSSION THREAD

**Discussion by mashed-potatoes** [kitsuned/PotatoNV#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by b1and** [comment]
May I ask why kirin 980 isn't supported, and is there a possibility for it to happen in the future?

**Reply by mashed-potatoes** [reply]
I doubt it since I don't have the right equipment and knowledge of new SoCs.
I recommend using HCU Client & DC-Phoenix.

**Comment by DJArty** [comment]
Hi!
What about Honor 8X support? (Kirin 710)

**Comment by 5tr0k3** [comment]
Hi, 
I am stroke, I have not done any coding regarding this but I do have some coding experience. 

I like the initiative and program very much and would like to work on the device that I have (MatePad T 8 (KOB2-L09)) to see if I can get it added to the list.
